src/tasks/MonoTargetsTasks/MarshalingPInvokeScanner/MarshalingPInvokeScanner.cs

```diff
@@ -104,6 +104,12 @@ private void ResolveInconclusiveTypes(HashSet<string> incompatible, string assyP
             }
         }
 
+        private bool IsDisableRuntimeMarshallingAttribute (MetadataReader mdtReader, StringHandle ns, StringHandle name)
+        {
+            return mdtReader.GetString(ns) == "System.Runtime.CompilerServices" &&
+                   mdtReader.GetString(name) == "DisableRuntimeMarshallingAttribute";
```

Comment:
This will avoid allocating strings.

```suggestion
            return mdtReader.StringComparer.Equals(ns, "System.Runtime.CompilerServices") &&
                   mdtReader.StringComparer.Equals(name, "DisableRuntimeMarshallingAttribute");
```